crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -86,7 +86,7 @@ pub struct RollupId {
         feature = "serde",
         serde(serialize_with = "crate::serde::base64_serialize")
     )]
-    inner: [u8; 32],
+    pub inner: [u8; 32],
```

Comment:
In general avoid exposing things as pub without good reason. With changes above we should be able to keep this non pub by using the existing functions.

Reply:
Got it, I'll be sure to try to look for more ways around it in the future. Should be ready for re-review now. Thanks for the feedback!